source/installation/install-with-docker.txt

```diff
@@ -0,0 +1,142 @@
+.. _rm-install-docker:
+
+===================
+Install with Docker
+===================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install and run Relational Migrator as a Docker container. 
+Running Relational Migrator with Docker ensures an installation process 
+that is both independent of the operating system and straightforward 
+to replicate.
+
+About this Task
+---------------
+
+You can also use `Docker Compose <https://docs.docker.com/compose/>`__ 
+to run Relational Migrator. You can find a Docker Compose example file
+in the 
+`download center <https://www.mongodb.com/try/download/relational-migrator>`__.
+
+.. #TODO# Update the link, when the compose file is added. 
+
+Before you Begin
+----------------
+
+Install `Docker <https://docs.docker.com/install/>`__
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Pull the MongoDB Docker Image
+      
+      .. code-block:: sh
+
+         docker pull public.ecr.aws/v4d7k6c9/relational-migrator:latest
+
+   .. step:: Run the Image as a Container
+      
+      The tabs below show various docker commands that are useful for 
+      running Relational Migrator:
+
+      .. tabs::
+
+         .. tab:: Basic
+            :tabid: basic-run
+
+            The ``-p 8080:8080`` in this command maps the container port to the host port.
+            This allows you to connect to Relational Migrator by browsing to 
+            `<http://localhost:8080>`__:
+
+            .. code-block:: sh
+
+               docker run -p 8080:8080 public.ecr.aws/v4d7k6c9/relational-migrator:latest
```

Comment:
It would be nice to name the container in each of the examples, e.g.
```
docker run --name mongodb-relational-migrator -p 8080:8080 public.ecr.aws/v4d7k6c9/relational-migrator:latest
```